src/media/css/base.styl

```diff
@@ -154,5 +154,42 @@ body {
 }
 
 #site-footer {
+    type--footer();
+    align-items: center;
     background: $color--greyscale--white;
+    display: flex;
+    justify-content: center;
+    padding: 20px;
+    
+    // Remove from the document flow and hide. Can't use display: none due to
+    // https://github.com/tj/nib/issues/291
+    position: absolute;
+    visibility: hidden;
+    +mq--show-footer() {
+        position: relative;
+        visibility: visible;
+    }
+
```

Comment:
the problem is what one person thinks should be grouped may not be clear to another on why they're grouped. it's just easier to have no seemingly-arbitrary whitespace. they're all within a single selector, .site-footer, which is enough of a group.